.editorconfig

```diff
@@ -6,9 +6,9 @@ end_of_line = lf
 indent_size = tab
 indent_style = space
 insert_final_newline = true
-max_line_length = 150
+max_line_length = 166
```

Comment:
I would use 120 and format (or ignore line with detekt comment)

Reply:
@e5l How do we decide on a number? 120 is the default for `detekt`, but I think it's too small (for my screen at least).

Reply:
Changing it from 166 to 120 will add `detekt` `MaxLineLength` warnings to more than 40 files and will break the build, so this will need to be done in a separate PR after this one is merged.